contrib/pkg/installmanager/installmanager.go

```diff
@@ -175,12 +178,34 @@ func (m *InstallManager) Run() error {
 
 	m.waitForInstallerBinaries()
 
-	dstInstallConfig := filepath.Join(m.WorkDir, "install-config.yaml")
-	m.log.Debugf("copying %s to %s", m.InstallConfig, dstInstallConfig)
-	err = m.copyFile(m.InstallConfig, dstInstallConfig)
+	// Generate an install-config.yaml:
+	sshKey, err := controllerutils.LoadSecretData(m.DynamicClient, cd.Spec.SSHKey.Name,
```

Comment:
Both ssh key and pull secret should be available as env vars when this runs, should we not use that?

Reply:
IIRC we kept those because they seemed to still be needed when we switched from env vars to install config to configure the installer. Looking now I can't see any hits for either OPENSHIFT_INSTALL_SSH_PUB_KEY or OPENSHIFT_INSTALL_PULL_SECRET so it looks to me like those are dead vars and we should probably kill them off.

Reply:
Pushed a change to remove those and fix the above, will see if it passes CI.